app/ts/components/AsyncText.tsx

```diff
@@ -0,0 +1,17 @@
+import { ComponentChildren } from 'preact'
+import { getRandomNumberBetween, removeNonStringsAndTrim } from '../library/utilities.js'
+
+export type AsyncTextProps = {
+	placeholderLength?: number
+	children?: ComponentChildren
+	class?: string
+}
+
+export const AsyncText = ({ placeholderLength = getRandomNumberBetween(10, 20), children, class: classNames }: AsyncTextProps) => {
```

Comment:
If you make the placeholderLength required then you don't need to generate random amount.
Does the length generated also not affect how the design looks? I would assume setting a fixed amount would be better to control the design.

Reply:
Would be going with this on #84